### PR TITLE
Collect raw results from scale runs.

### PR DESCRIPTION
**Pull request by jayhawk87**
Raw results are needed for determining performance
regressions via hypothesis testing.

Signed-off-by: Ryan Moats rmoats@us.ibm.com

### CODE REVIEW

**Comment by huikang** [comment]
LGTM

File: ci/scale-run.sh [hunk 1]
```diff
@@ -64,25 +64,33 @@ $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
 $OVNSUDO docker exec ovn-rally rally-ovs task start /root/rally-ovn/workload/create_networks.json
 TASKID=$($OVNSUDO docker exec ovn-rally rally task list --uuids-only)
 $OVNSUDO docker exec ovn-rally rally task report $TASKID --out /root/create-networks-output.html
+$OVNSUDO bash -c "docker exec ovn-rally rally task results $TASKID > /root/create-networks-data.json"
```
Comment (mestery) [inline ci/scale-run.sh]: This is wrong, it should be:

OVNSUDO docker exec ovn-rally rally task results $TASKID --out /root/create-networks-data.json

File: ci/scale-run.sh [hunk 2]
```diff
@@ -64,25 +64,33 @@ $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
...
 $OVNSUDO docker cp ovn-rally:/root/create-networks-output.html .
+$OVNSUDO docker cp ovn-rally:/root/create-networks-data.json .
 $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
 
 $OVNSUDO docker exec ovn-rally rally-ovs task start /root/rally-ovn/workload/create_and_list_lports.json
 TASKID=$($OVNSUDO docker exec ovn-rally rally task list --uuids-only)
 $OVNSUDO docker exec ovn-rally rally task report $TASKID --out /root/create-and-list-lports-output.html
+$OVNSUDO bash -c "docker exec ovn-rally rally task results $TASKID > /root/create-and-list-lports-data.json"
```
Comment (mestery) [inline ci/scale-run.sh]: This is wrong, it should be:

OVNSUDO docker exec ovn-rally rally task results $TASKID --out /root/create-networks-data.json

File: ci/scale-run.sh [hunk 3]
```diff
@@ -64,25 +64,33 @@ $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
...
 $OVNSUDO docker cp ovn-rally:/root/create-and-list-lports-output.html .
+$OVNSUDO docker cp ovn-rally:/root/create-and-list-lports-data.json .
 $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
 
 $OVNSUDO docker exec ovn-rally rally-ovs task start /root/rally-ovn/workload/create_and_list_acls.json
 TASKID=$($OVNSUDO docker exec ovn-rally rally task list --uuids-only)
 $OVNSUDO docker exec ovn-rally rally task report $TASKID --out /root/create-and-list-acls-output.html
+$OVNSUDO bash -c "docker exec ovn-rally rally task results $TASKID > /root/create-and-list-acls-data.json"
```
Comment (mestery) [inline ci/scale-run.sh]: This is wrong, it should be:

OVNSUDO docker exec ovn-rally rally task results $TASKID --out /root/create-networks-data.json

File: ci/scale-run.sh [hunk 4]
```diff
@@ -64,25 +64,33 @@ $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
...
 $OVNSUDO docker cp ovn-rally:/root/create-and-list-acls-output.html .
+$OVNSUDO docker cp ovn-rally:/root/create-and-list-acls-data.json .
 $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
 
 $OVNSUDO docker exec ovn-rally rally-ovs task start /root/rally-ovn/workload/create_and_bind_ports.json
 TASKID=$($OVNSUDO docker exec ovn-rally rally task list --uuids-only)
 $OVNSUDO docker exec ovn-rally rally task report $TASKID --out /root/create-and-bind-ports-output.html
+$OVNSUDO bash -c "docker exec ovn-rally rally task results $TASKID > /root/create-and-bind-ports-data.json"
```
Comment (mestery) [inline ci/scale-run.sh]: This is wrong, it should be:

OVNSUDO docker exec ovn-rally rally task results $TASKID --out /root/create-networks-data.json

**Comment by mestery** [comment]
This needs to be fixed before it merges.

File: ci/scale-run.sh (hunk 1 above)
Comment (mestery) [inline ci/scale-run.sh]: The redirection is the issue, try adding "--out /root/create-networks-data.json"

**Comment by mestery** [comment]
Closing in lieu of PR #68.

**Pull request closed without merging**